interface.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function() {
+  var game = new Game();
+
+  $('.User_Roll').change(function() {
+    let pins = parseInt($('.User_Roll').val());
+    $(`#f${game.frameCount()}r${game.rollCount()}`).text(pins);
+    if (game.rollCount() === 1 && pins === 10) {
+      $(`#f${game.frameCount()}r2`).text("X");
+    }
```

Comment:
I think extracting these lines of code into a separate clearly-named function would make this method a bit more readable